src/main/app/src/components/ReduxForm/index.tsx

```diff
@@ -1,17 +1,21 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 
 import { reduxForm } from 'redux-form';
 
-import FormNameContext from '#/src/contexts/FormNameContext';
+import FormContext from '#/src/contexts/FormContext';
 
-const ReduxForm = reduxForm({ initialValues: {}, enableReinitialize: true })(
-  ({ children, form }) => {
-    return (
-      <FormNameContext.Provider value={form}>
-        {children}
-      </FormNameContext.Provider>
-    );
-  }
-);
+const ReduxFormWrapper = reduxForm({
+  initialValues: {},
+  enableReinitialize: true,
+})(({ children }) => <>{children}</>);
+
+const ReduxForm = ({ form, disabled = false, children, initialValues }) => {
+  const formCtx = useMemo(() => ({ name: form, disabled }), [form, disabled]);
+  return (
+    <ReduxFormWrapper form={form} initialValues={initialValues}>
```

Comment:
Mites tää menee, jos rivillä 7 on jo kutsuttu reduxFormia jollain initialValuesilla niin tekeekö tässä annettu initialValues mitään? Eli meneekö tää propsi vaan tolle lambdarendererille rivillä 10 joka vaan syö sen initialValues propsin

Reply:
Eikun näinhän se on ollut aiemminkin, pitäshän sen sitten olla kuosis

Reply:
Jep, eli muutin tähän vain tuota rakennetta hieman, että saan FormContext:in tuota kautta kaikille eikä tule tyyppivirheitä.